components/Image/image.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Vrushabh Dhatrak" />
+    <meta name="description" content="The Image component provides different states of Image responsiveness." />
+    <link rel="stylesheet" href="./../../styles/main.css" />
+    <script src="./../../script/header.js" defer></script>
```

Comment:
Is it correct use of defer?? Or we can use it in bottom for better ?

Reply:
Because defer work as an async it does not matter writing script anywhere in the entire file